ext-php-rs-derive/src/lib.rs

```diff
@@ -125,3 +127,15 @@ pub fn php_extern(_: TokenStream, input: TokenStream) -> TokenStream {
     }
     .into()
 }
+
+// TODO(david): potentially add option to ignore field in exchange for `Default::default()`?
```

Comment:
Is this needed? I'm not sure if I see the purpose of having a field with a default value but it seems simple enough to add.